config/jobs/kubernetes/generated/generated.yaml

```diff
@@ -25,7 +25,10 @@ periodics:
       - --timeout=180m
       - --env=KUBE_PROXY_DAEMONSET=true
       - --env=ENABLE_POD_PRIORITY=true
-      - --env=KUBE_FEATURE_GATES=AllAlpha=true,InTreePluginGCEUnregister=false,DisableCloudProviders=false,DisableKubeletCloudCredentialProviders=false
+      # TODO: Fix generate_tests.py to set DisableCloudProviders,DisableKubeletCloudCredentialProviders to true when --cloud-proider=external
+      # And also add --env=ENABLE_AUTH_PROVIDER_GCP=true
+      - --env=KUBE_FEATURE_GATES=AllAlpha=true,InTreePluginGCEUnregister=false,DisableCloudProviders=true,DisableKubeletCloudCredentialProviders=true
+      - --env=ENABLE_AUTH_PROVIDER_GCP=true
```

Comment:
I didn't add `--env=CLOUD_PROVIDER_FLAG=external` as the test initialisation scripts by default set it to `external`.